examples/retrieving-results.ipynb

```diff
@@ -0,0 +1,873 @@
n', record_id='18435512', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435513', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435514', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435515', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435516', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435517', cmiles='[H:18][c:1]1[c:2]([c:7]([c:9]([n:12][c:4]1[H:21])[O:16][H:24])[c:11]2[n:13][c:10]([n:14][o:15]2)[c:8]3[c:3]([c:5]([s:17][c:6]3[H:23])[H:22])[H:20])[H:19]', inchi_key='MLMBZQLCTHUGAO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435518', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12][C:8]([H:18])([H:19])/[C:7](=[N:9]/[O:11][H:22])/[N:10]([H:20])[H:21])[Cl:14])[H:17])[Cl:13])[H:16]', inchi_key='BTOTVAISQCMDLO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435519', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12][C:8]([H:18])([H:19])/[C:7](=[N:9]/[O:11][H:22])/[N:10]([H:20])[H:21])[Cl:14])[H:17])[Cl:13])[H:16]', inchi_key='BTOTVAISQCMDLO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435520', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12][C:8]([H:18])([H:19])/[C:7](=[N:9]/[O:11][H:22])/[N:10]([H:20])[H:21])[Cl:14])[H:17])[Cl:13])[H:16]', inchi_key='BTOTVAISQCMDLO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435521', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12][C:8]([H:18])([H:19])/[C:7](=[N:9]/[O:11][H:22])/[N:10]([H:20])[H:21])[Cl:14])[H:17])[Cl:13])[H:16]', inchi_key='BTOTVAISQCMDLO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435522', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12][C:8]([H:18])([H:19])/[C:7](=[N:9]/[O:11][H:22])/[N:10]([H:20])[H:21])[Cl:14])[H:17])[Cl:13])[H:16]', inchi_key='BTOTVAISQCMDLO-UHFFFAOYSA-N'), OptimizationResult(type='optimization', record_id='18435523', cmiles='[H:15][c:1]1[c:2]([c:5]([c:3]([c:6]([c:4]1[O:12]
```

Comment:
I think this should be `n_molecules` and we should see a difference in the numbers if the molecule deduplication is working.